app/models/movie.rb

```diff
@@ -3,39 +3,35 @@
 class Movie < ApplicationRecord
   has_many :parties, dependent: :destroy
 
-  def release_year(release_date)
-    if release_date != ''
-      Date.parse(release_date).year
-    else
-      return 'n/a'
-    end
-  end
+  # def release_year(release_date)
+  #   if release_date != ''
+  #     Date.parse(release_date).year
+  #   else
+  #     return 'n/a'
+  #   end
+  # end
 
-  def display_genres(genre_array)
-    genre_array.map do |genre|
-      genre[:name]
-    end
-  end
+  # def display_genres(genre_array)
+  #   genre_array.map do |genre|
+  #     genre[:name]
+  #   end
+  # end
 
-  def total_runtime(runtime)
-    hours = runtime / 60
-    remaining = runtime % 60
-    "#{hours}h #{remaining}m"
-  end
+  # def total_runtime(runtime)
```

Comment:
We need to look into a time_to_words function built into Rails. I messed around with this for maybe 30 minutes, and was only able to get it to say 'About 3 hours'.